### DISCUSSION TITLE
from foam to vnote

### DISCUSSION THREAD

**Discussion by ilkosta** [tamlok/vnote#2486] [General]
Hello,
I currently organize my notes via [foam](https://foambubble.github.io/) but was happily surprised to discover vnote. 

I would like to experiment with vnote because I particularly appreciate its lightness and immediacy, even if in vscode/foam I have a greater number of plugins available that add functionality.

I would like to be able to use vnote to continue editing the notes that I already have organized in directories, and I was wondering if it is possible to insert links in the form of wikilinks to documents alread inside the notebook.

Thanks for any suggestions

**Comment by tamlok** [comment]
Welcome! Just copy the path of your target file and paste to the note in VNote. VNote will prompt you how to handle the link in MD.
